.github/workflows/goproxy-warmup.yml

```
@@ -0,0 +1,15 @@
+on:
+  release:
+    types:
+      - created
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "**/v[0-9]+.[0-9]+.[0-9]+"
```

```suggestion
      - "v\d+\.\d+\.\d+"
      - "**/v\d+\.\d+.\d+"
```
statt [0-9] geht `\d`
und punkte muss man in regex escapen. also vorausgesetzt das ist ein regex.

---

hab das einfach aus der Doku kopiert und den code nicht weiter analysiert